### PR TITLE
fix: geometry edit tools and styling

### PR DESCRIPTION
**Pull request by AmandaTamanda**
Sets geometry edit tools as inactive until activated by the users and applies styling fixes.

### CODE REVIEW

File: src/components/EditFeatureDrawer/EditFeatureGeometryToolbar/index.tsx [hunk 1]
```diff
@@ -221,10 +222,16 @@ export const EditFeatureGeometryToolbar: React.FC<EditFeatureGeometryToolbarProp
         className="geometry-edit-tb"
         alignment="vertical"
       >
-        <ToggleGroup>
+        <ToggleGroup
+        selected={selected}
+        onChange={(_evt: any, value: string | undefined) => {
+          setSelected(value)
+        }}
+        >
```
Comment (KaiVolland) [inline src/components/EditFeatureDrawer/EditFeatureGeometryToolbar/index.tsx:230]:
```suggestion
        <ToggleGroup
          selected={selected}
          onChange={(_evt: any, value: string | undefined) => {
            setSelected(value)
          }}
        >
```

File: src/components/EditFeatureDrawer/EditFeatureGeometryToolbar/index.tsx [hunk 2]
```diff
@@ -221,10 +222,16 @@ export const EditFeatureGeometryToolbar: React.FC<EditFeatureGeometryToolbarProp
...
           {
             allowedEditMode.includes('CREATE') ?
               <DrawButton
+                value="one"
```
Comment (KaiVolland) [inline src/components/EditFeatureDrawer/EditFeatureGeometryToolbar/index.tsx:234]: IMHO a more speaking value like "CREATE" would be better. Same for the buttons below.

**Review by KaiVolland — COMMENTED** [review]
Very nice. It's always nice to have a screenshot when styling changes if possible.

**Comment by AmandaTamanda** [comment]
> Very nice. It's always nice to have a screenshot when styling changes if possible.

thanks for the review, the styling changes were more about restoring old styles which were no longer available due to the updates in the next version. I still generally looks the same.

**Review by KaiVolland — APPROVED** [review]

**Review by dnlkoch — APPROVED** [review]
Very nice, thanks!

**Comment by AmandaTamanda** [comment]
> Very nice, thanks!

Thanks for the review!

**Pull request merged**